### PR TITLE
Added enhancements/bug fixes

### PR DESCRIPTION
**Pull request by shahrizal-binary**
Hi Vitaly,

I made two small changes to your codes.
1. Handling of Multiple FIX messages within one single TCPIP message.

Example: 

---

Received FIX message:
## 8=FIX.4.4|9=112|35=W|34=60|49=FixServer|52=20151216-07:38:59.438|56=CLIENT1|55=CADCHF|268=2|269=0|270=0.71849|269=1|270=0.71861|10=233|8=FIX.4.4|9=112|35=W|34=61|49=FixServer|52=20151216-07:38:59.438|56=CLIENT1|55=NZDUSD|268=2|269=0|270=0.67527|269=1|270=0.67535|10=042|

In this sample TCP/IP message I received from feed provider, it contains two FIX messages for Market Data. What I did was I split this into each single FIX message before parsing them.
1. Handling of Grouped Information.

In the sample message above, the grouped info is the Tag 268(NoMDEntries). 

268=2|269=0|270=0.67527|269=1|270=0.67535

It contains a repeated tags of 269 and 270 which stands for MDEntryType and MDEntryPx.

Here is a simple example of handler on how to traverse the grouped information:

---

  sub handler {
     my %resp = @_;

```
 while ( my ( $k, $v ) = each(%resp) ) {
    print "Key: ".$k." \n";
    if ( ref($v) eq 'ARRAY' ) {
            print "Value is an array\n";
            for my $g (@$v) {
                    print "ArrayValue1 : ".$g->{MDEntryType}."\n";
                    print "ArrayValue2 : ".$g->{MDEntryPx}."\n";
            }
    } else {
                    print "Value: ".$v."\n";
    }

 }

 return 1;
```

  }

---

Thanks,
Shahrizal

### CODE REVIEW

File: lib/FIX/Lite.pm [hunk 1]
```diff
@@ -2,7 +2,7 @@ package FIX::Lite;
 
 use vars qw($VERSION @ISA);
 use warnings;
-use strict;
+#use strict;
```
Comment (jy-deriv) [inline lib/FIX/Lite.pm]: why did you remove use strict?

Comment (shahrizal-binary) [inline lib/FIX/Lite.pm]: I commented it out temporarily yesterday while debugging. I have
uncommented it back.
